cellpack/bin/simularium_converter.py

```diff
@@ -202,11 +204,15 @@ def get_ingredient_data(self, cytoplasm, main_container, ingredient):
         return (ingredient_name, cytoplasm_data, container_data)
 
     def get_euler_from_matrix(self, data_in):
-        rotation_matrix = [data_in[0][0:3], data_in[1][0:3], data_in[2][0:3]]
-        return R.from_matrix(rotation_matrix).as_euler("XYZ", degrees=False)
+        # rotation_matrix = [data_in[0][0:3], data_in[1][0:3], data_in[2][0:3]]
+        # euler = R.from_matrix(rotation_matrix).as_euler("XYZ", degrees=False)
+        euler2 = TR.euler_from_matrix(data_in)
+        return euler2
 
     def get_euler_from_quat(self, data_in):
-        return R.from_quat(data_in).as_euler("XYZ", degrees=False)
+        # euler = R.from_quat(data_in).as_euler("XYZ", degrees=False)
+        euler2 = TR.euler_from_quaternion(data_in)
+        return euler2
```

Comment:
with these new changes the HIV rotations no longer look right. I would prefer to have the HIV/PBD working then trying to get blender OBJs to work because from the cellPACK chart blender is the most annoying export anyways

Reply:
(also, just a note on commits..... remember to just write the code as you want it (`euler` instead of `euler2` and don't comment out code, just delete it). It's easy enough to revert commits if we don't want the change and if we do want it we dont have to do the extra work of cleaning up the code later, or being confused why it looks this way.